### PR TITLE
Add warning when interpolating to a boundary that contains multiple polylines

### LINKED ISSUE #370: Add warning when interpolating to a boundary that contains multiple polylines
A warning should be added when interpolating to a boundary that contains multiple polylines in one *.pli-file. See https://issuetracker.deltares.nl/browse/UNST-7012

### PR DESCRIPTION
**Pull request by Tammo-Zijlker-Deltares**
Fixes #370

### CODE REVIEW

**Comment by veenstrajelmer** [comment]
Currently the warning is raised by an example script. Could you instead put this warning in the used function? I believe in case of tide and other models like CMEMS, all processes pass trough `interp_regularnc_to_plipoints()`.

File: tests/examples_workinprogress/workinprogress_modelbuilder.py [hunk 1]
```diff
@@ -135,6 +135,13 @@
 #TODO: two waterlevelbnds need to share same physical plifile in order to + (https://issuetracker.deltares.nl/browse/UNST-5320).
 list_quantities = ['salinitybnd','temperaturebnd','uxuyadvectionvelocitybnd','waterlevelbnd','tide'] #TODO: JV1 stable ("WARNING: Boundary link 00000468 already claimed")
 #list_quantities = ['waterlevelbnd','salinitybnd','temperaturebnd','uxuyadvectionvelocitybnd','tide'] #TODO: JV6 "ERROR  : update_ghostboundvals: not all ghost boundary flowlinks are being updated"
+
+#Check whether the polyfile contains multiple polyline, in that case show a warning
+pli = hcdfm.PolyFile(poly_file)
+if len(pli.objects) > 1:
+    print(f"Warning: the polyfile {poly_file} contains multiple polylines. Only the first one will be used by DFLOW-FM for the boundary conditions.")
```
[the remark this answers is not in the document]
Comment (Tammo-Zijlker-Deltares) [reply tests/examples_workinprogress/workinprogress_modelbuilder.py]: I have changed this to a UserWarning

File: tests/examples_workinprogress/workinprogress_modelbuilder.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 import xarray as xr
 import pandas as pd
 import contextily as ctx
-
+import warnings
```
Comment (veenstrajelmer) [inline tests/examples_workinprogress/workinprogress_modelbuilder.py:17]: Minor thing, but please re-introduce the whitespace below the imports in this script

**Comment by Tammo-Zijlker-Deltares** [comment]
Unfortunately I could not undo the commits (otherwise the undoing of commits again shows up). I have removed all changes from workinprogress_modelbuilder

**Review by veenstrajelmer — APPROVED** [review]
All previous comments were processed
